Lundalogik/lime-elements
# feat(limel-text-editor): remove native url validation from the link input field

Fixes Lundalogik/crm-feature#4166

## How to Test

Go into one of the limel prosemirror adapter examples and add a link. Now try writing "test" -> hit tab -> writing "test". Then click outside the input field and note that it does not turn red (before the problem was that a phrase like "test" was not considered a valid URL).

## Review:

### Browsers tested:
(Check any that applies, it's ok to leave boxes unchecked if testing something didn't seem relevant.)

Windows:

Linux:

macOS:

Mobile:
